From 3f5a205391d01edb29f4dbd04eb46c7f181e21d1 Mon Sep 17 00:00:00 2001
From: Cold Fry <coldfry.business@gmail.com>
Date: Thu, 2 Apr 2026 23:16:17 +0100
Subject: [PATCH 7/9] prettier!

---
 src/transpilation/resolve.ts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/transpilation/resolve.ts b/src/transpilation/resolve.ts
index d4043ef..0aba5b2 100644
--- a/src/transpilation/resolve.ts
+++ b/src/transpilation/resolve.ts
@@ -41,8 +41,7 @@ class ResolutionContext {
     ) {
         const unique = [...new Set(options.noResolvePaths)];
         this.noResolvePaths = unique.map(x => picomatch(x));
-        this.pathsBase =
-            options.baseUrl ?? (options.configFilePath ? path.dirname(options.configFilePath) : undefined);
+        this.pathsBase = options.baseUrl ?? (options.configFilePath ? path.dirname(options.configFilePath) : undefined);
     }
 
     public addAndResolveDependencies(file: ProcessedFile): void {
-- 
2.50.1 (Apple Git-155)

